index.html

```diff
@@ -20,7 +20,7 @@
     title: "Tutorial"
     excerpt: "TermIt solves various problems, from creating vocabularies to annotating resources."
     url: "/tutorial"
-    btn_label: "Learn me how"
+    btn_label: "Tell me how"
```

Comment:
Show :-) ?

Reply:
definitely tell, show is in the second article